jwst/lastframe/lastframe_sub.py

```diff
@@ -12,36 +12,36 @@
 
 def do_correction(output):
     """
-    Short Summary
-    -------------
-    The sole correction is to reset to DO_NOT_USE the GROUP data quality flags
-    for the final group, if the number of groups is greater than 2.
+    MIRI correction to set data quality flag of final group to DO_NOT_USE.
```

Comment:
Rephrase as a command, e.g.:
```suggestion
    Set data quality flag of final groups to DO_NOT_USE.
```